Management/src/ActuatorWeb/Pages/Weather.cshtml.cs

```diff
@@ -0,0 +1,61 @@
+using Microsoft.AspNetCore.Http.Extensions;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace Steeltoe.Samples.ActuatorWeb.Pages;
+
+public class WeatherModel(IHttpClientFactory httpClientFactory, ILogger<WeatherModel> logger) : PageModel
+{
+    public required WeatherForecast[] Forecasts { get; set; } = [];
+    internal string? ForecastStartDate { get; set; }
+    internal string DaysCountSelected { get; set; } = string.Empty;
+
+    public async Task OnGet(string? fromDate, int? daysCount)
+    {
+        // Log messages at various levels for loggers actuator demonstration
+        logger.LogCritical("Test Critical message");
+        logger.LogError("Test Error message");
+        logger.LogWarning("Test Warning message");
+        logger.LogInformation("Test Informational message");
+        logger.LogDebug("Test Debug message");
+        logger.LogTrace("Test Trace message");
+
+        using var httpClient = httpClientFactory.CreateClient(nameof(WeatherModel));
+        try
```

Comment:
See https://github.com/SteeltoeOSS/Samples/commit/6bddce88a8bb3dc91a21cbdb3bd4a432ac776e5c.

This follows the [recommended pattern](https://learn.microsoft.com/en-us/dotnet/architecture/microservices/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests#implement-your-typed-client-classes-that-use-the-injected-and-configured-httpclient), particularly: inject `HttpClient` instead of the factory. But doing so directly into `WeatherModel` doesn't work, due to the way Razor registers models in the IoC container. See [here](https://stackoverflow.com/questions/65777953/why-httpclient-does-not-hold-the-base-address-even-when-its-set-in-startup) for details.

I also switched to `List<>` instead of array, because that's what application developers are more familiar with.

Reply:
Consider changing `Security\src\AuthWeb\Controllers\HomeController.cs` likewise.

Reply:
`AuthWeb` uses two different `HttpClient`s, so I'm not sure it still makes so much sense there... I think it's a distraction from the purpose of this PR, but worth a follow-up so I opened #345